CONTRIBUTING.md

```diff
@@ -36,6 +36,8 @@ Ruby version.
 tests similarly to TeamCity. The database is destroyed between each
 test file.)
 
+Note that the actual CI test suit is ran with [this script](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/tests/activerecord.go). And test failures are tracked [cockroach#97283](https://github.com/cockroachdb/cockroach/issues/97283)
```

Comment:
The CI test suite on the adapter repo is not run with that script, it's for running the nightly roachtest run of the activerecord test in the [cockroach](https://github.com/cockroachdb/cockroach/) repo.

I would also not hardcode the issue number because this is only the current issue tracking the master branch. Once the tests are fixed, it will be closed and a new issue will be opened for future test failures.

Reply:
That's right - the adapter CI uses this script actually: https://github.com/cockroachdb/activerecord-cockroachdb-adapter/blob/002b0973e68211bbbe3f112f669b2c91a555ea4a/build/teamcity-test.sh#L71-L74

This is defined in the TeamCity build job: https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_ScratchProjectPutTcExperimentsInHere_ActiveRecordTestSuite?mode=builds

As a future project, it would be simpler to execute the CI using Github actions (which wasn't available at the time when we first set this project up). This would make it simpler so we don't have to go to a different place to configure CI for this project. I made an issue: https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/279

As far the where the test failures are tracked: for our nightlies, this is the place we can link to: https://github.com/cockroachdb/cockroach/blob/8d10f15db8e5efaff57ec332c438211895c6c608/pkg/cmd/roachtest/tests/activerecord_blocklist.go#L36

The ideal state is that none of them should fail - either we skip them if we determine it cannot work with CRDB, or we fix the bug in the tests so it passes. I don't think we need to link to this test tracker here.